wg-batch/annual-report-2024.md

```diff
@@ -2,32 +2,136 @@
+
+Deployment/StatefulSet integration provides an avenue for the usage of Kueue for serving workloads. Serving leads to a need for sharing/preemption of model servers that may leverage accelerators. Kueue provides an integration with popular methods of deploying services (Deployment/StatefulSet).
+
+##### JobSet
+
+Jobset has had 4 release in 2024.
+
+- [Release 0.4](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.4.0)
+
+- [Release 0.5](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.5.0)
+
+- [Release 0.6](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.6.0)
+
+- [Release 0.7](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.7.0)
+
+A major achievement of JobSet has been the adoption of JobSet as a component for Kubeflow Training Operator V2.
+There has been a collaborative effort with the Kubeflow community and the batch community to implement the features needed for this integration.
+
+[Metaflow](https://github.com/Netflix/metaflow/pull/1804) has adopted the use of JobSet for distributed ML training.
+
+##### KJob
+
+[KJob](https://github.com/kubernetes-sigs/kjob?tab=readme-ov-file#kjob) has been started to provide a CLI friendly way for users to submit batch jobs.
+The HPC/ML community tend to prefer CLI over YAML so the focus was to provide a templated solution for submitting batch jobs.
+Another focus of this project is to provide a smooth transition for Slurm users.
+
+#### KEPs
+
+WG-Batch provided a series of kubernetes enhancements that improved the experience of batch workloads on Kubernetes. In 2024, this group proposed/implemented the following KEPs.
+
+- [Job Managed By](https://github.com/kubernetes/enhancements/issues/4368)
+  - Promoted to beta.
+
+- [Job Success Policy](https://github.com/kubernetes/enhancements/issues/3998)
+  - Promoted to beta.
+
+- [Elastic Index Jobs](https://github.com/kubernetes/enhancements/issues/3715)
+  - Promoted to stable.
+
+- [Pod Failure Policy](https://github.com/kubernetes/enhancements/issues/3329)
+  - Promoted to stable.
+
+- [Pod Index Label](https://github.com/kubernetes/enhancements/issues/4017)
+  - Promoted to stable.
+
+### Talks
+
+- WG-Batch Update at Kubecon
+  - Authors: Kevin Hannon and Marcin Wielgus
+  - Kubecon NA, Salt Lake City
+
+- Keynote: MultiCluster Batch Jobs Dispatching with Kueue at CERN
+  - Authors: Ricardo Rocha and Marcin Wielgus
+  - Kubecon NA, Salt Lake City
+
+- Multitenancy and Fairness at Scale with Kueue: A Case Study
+  - Authors: Aldo Culquicondor and Rajat Phull
+  - Kubecon NA, Salt Lake City
+
+- Advanced Resource Management for Running AI/ML Workloads with Kueue
+  - Authors: Michał Woźniak and Yuki Iwai
+  - Kubecon EU, Paris
+
+- Scale Your Batch / Big Data / AI Workloads Beyond the Kubernetes Scheduler
+  - Authors: Antonin Stefanutti and Anish Asthana
+  - KubeCon EU, Paris
+
+- WG-Batch Update
+  - Author: Marcin Wielgus
```

Comment:
```suggestion
  - Author: Michał Woźniak and Yuki Iwai
```

:)  https://kccnceu2024.sched.com/event/1Yhhw

Reply:
sorry about that. I'm not sure where I saw that now..

Reply:
I don't care it :)